src/App.tsx

```diff
@@ -1,14 +1,69 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoId, setTodoId] = useState(0);
+  const [userId, setUserId] = useState(0);
+  const [selectedOption, setSelectedOption] = useState('all');
+  const [query, setQuery] = useState('');
+
+  const getTodoById = (id: number) => {
+    return todos.find(todo => todo.id === id) as Todo;
+  };
+
+  const selectTodoId = (id: number) => {
+    setTodoId(id);
+  };
+
+  const selectUserId = (id: number) => {
+    setUserId(id);
+  };
```

Comment:
these are generally redundant
unless you pursue to shorten the onClick prop drilling
and yeah, you do

then you can at least single-line such :)
```suggestion
  const selectTodoId = (id: number) => { setTodoId(id) };
  const selectUserId = (id: number) => { setUserId(id) };
```